atomic_reactor/dirs.py

```diff
@@ -24,6 +24,15 @@ class DockerfileNotExist(Exception):
     """Dockerfile does not exist."""
 
 
+class BuildDirsNotInitialized(Exception):
+    """Build directories are not initialized."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            "Build directories are not initialized inside the root build directory."
+        )
```

Comment:
It's a bit weird that the args and kwargs are just ignored. Maybe the init shouldn't have any args? Or, instead of overriding init, avoid duplicating the code some other way (constant, helper function...)?